src/status_im/js_dependencies.cljs

```diff
@@ -9,3 +9,4 @@
 (def text-encoding       (js/require "text-encoding"))
 (def js-sha3             (js/require "js-sha3"))
 (def web3-utils          (js/require "web3-utils"))
+(def keccak              (js/require "keccak"))
```

Comment:
Is there a way to avoid this dependency and rely on existing deps like web3 or web3-utils?

Reply:
@jeluard Yes, sorry!
Removed.